README.md

```diff
@@ -1,2 +1,52 @@
-# proposals
-Proposals for new Jupyter subprojects to enter into incubation
+# Proposals For Incubation on [jupyter-incubator](https://github.com/jupyter-incubator)
+
+
+## Overview
+
+Project Jupyter is organized as a set of Subprojects that are each a GitHub
+repository with a development team that follows the Jupyter governance, license and
+contribution model. Officially supported and maintained subprojects are hosted on
+the [jupyter](https://github.com/jupyter) GitHub orgnization.
+
+Incubation refers to the process of a Subproject initially being developed outside
+the official Jupypter organization. The
+[jupyter-incubator](https://github.com/jupyter-incubator) GitHub organization is
+maintained and managed by Project Jupyter as one possible location where Subproject
+can be incubated. The [New Subproject
+Process](https://github.com/jupyter/governance/blob/master/newsubprojects.md)
+document in the Jupyter [governance](https://github.com/jupyter/governance)
+repository details the process of incubation, the process of incorporation into the
+main Jupyter organization and the criteria for incorporation.
+
+
+## Incubation process
+
+This repository is used for incubation proposals. To initiate this process:
+
+1. Read the [New Subproject Process](https://github.com/jupyter/governance/blob/master/newsubprojects.md)
+   to see if incubation on [jupyter-incubator](https://github.com/jupyter-incubator) is appropriate for your new Subproject (it might not be). If is doubt, please send an email to the main Jupyter list asking for input.
+3. Pick a name for your new Subproject that is free of trademark problems.
```

Comment:
I would suggest adding some 'common sense and community guidance' here. It's not just legalese, but a couple of sentences about how to avoid treading on the names that other projects have picked, can help people avoid problems.

And, perhaps more importantly, we should clarify what *our* policy about names is going to be... People are inevitably going to come up with planet puns, variations on the Jupyter word, etc.  We should provide some guidance here.

I don't think we need to  hold off on merging the PR itself on this point, BTW, we can refine further over the next few days, just flagging the issue.